hal/inc/timer_hal.h

```diff
@@ -36,18 +36,23 @@
 
 /* Exported macros -----------------------------------------------------------*/
 
+#define HAL_Timer_Microseconds HAL_Timer_Get_Micro_Seconds
```

Comment:
I am unsure about moving the #define since it will change the actual exported function name. I realize the chances are small, but someone could be using the HAL_Timer_Get_Micro_Seconds function.  What is the value gained to moving the #define - if it's minimal, I'd prefer not to change it unless there is a known gain.

Reply:
hm, but these macros do the opposite: they define aliases for already exported functions, rather than define their exported names. I don't see a possibility to break anything with this change, but, for sure, it doesn't bring any value either :)